src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -45,6 +45,54 @@ Then, you can see the error in your dashboard:
 
 [{% asset js-index/error-message.png alt="Error in Unresolved Issues with title This is my fake error message" %}]({% asset js-index/error-message.png @path %})
 
+{% capture __alert_content -%}
+When you are using our CDN, it may happen, that some people will use ad-blocking or script-blocking extensions. When this happens, our SDK won't be fetched and initialized properly. Because of this, any call to SDKs API will fail and may cause your application to misbehave. If this could apply to your use-case, read the section below on how to mitigate this issue.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Dealing with Ad-Blockers"
+  content=__alert_content
+  level="warning"
+%}
+
+The best way to deal with script-blocking extensions is to use the SDK package directly through the `npm` and bundle it with your application.
+This way, you can be sure, that the code will be always there as you expect it to be.
+
+The second way is to download the SDK from our CDN and host it yourself, this way SDK will still be separated from the rest of your code, but you'll be certain that it won't be blocked, as it's origin will be the same as the origin of your website.
+
+You can easily fetch it using `curl` or any other similar tool:
+
+```sh
+$ curl https://browser.sentry-cdn.com/{% sdk_version sentry.javascript.browser %}/bundle.min.js -o sentry.browser.{% sdk_version sentry.javascript.browser %}.min.js -s
+```
+
+The last option, is to use `Proxy` guard, that will make sure that your code won't break, even when you will call our SDK which was blocked.
+`Proxy` is supported by all browser except Internet Explorer, however there are no extensions in these browsers anyway. Also if `Proxy` is not in any of your user's browser, it will be silently skipped, so you don't have to worry about it breaking anything.
```

Comment:
```suggestion
`Proxy` is supported by all browsers except Internet Explorer. However, there are no extensions in these browsers anyway. Also, if `Proxy` is not in any of your users' browsers, it will be silently skipped, so you don't have to worry about it breaking anything.
```